README.md

```diff
@@ -33,15 +33,16 @@ Under your database security settings, create a new Key with the **admin** role
 
 ### 3. Decide on your daily/weekly/monthly spend
 
-Next, you'll need to specify how often you want to buy Bitcoin, and how much you want to spend on each purchase. You do this by setting the following environment variables:
+Next, you'll need to specify how often you want to buy coins, how much you want to spend on each purchase, and the cryptocurrency pair you want to purchase. You do this by setting the following environment variables:
 
 1. `BUY_FREQUENCY`
     - `DAILY` - if you want to buy every day
     - **WEEKLY_[1-7]** - if you want to buy once a week. e.g. `WEEKLY_1` to buy every Monday
     - **MONTHLY_[1-29]** - if you want to buy once a month. e.g. `MONTHLY_5` to buy on the 5th day of each month
 2. `BUY_AMOUNT`
 The amount (in Naira) you want to purchase at each frequency.
-
+3. `BUY_CURRENCIES`
+The currency pair you want to trade. e.g. `btc_ngnt` to buy btc with ngnt. To see all the cryptocurrency pairs supported by Buycoins, you can use the [getPairs](https://developers.buycoins.africa/orderbook-trading/glossary) method.
```

Comment:
Did you actually look at the code? This isn't what they put in. It should be in this format - `BTC,ETH,USDT` for example